docs/docs/guides/tracking/faqs.md

```diff
@@ -148,3 +148,40 @@ pref_scorer = PreferenceScorer(other_model=model_b)
 evaluation = Evaluation(dataset=dataset, scorers=[pref_scorer])
 evaluation.evaluate(model_a)
 ```
+
+## Pricing FAQs
+
+The following section provides answers to pricing FAQs. For the latest information, see the [pricing page](https://wandb.ai/site/pricing/).
+
+### How is Weave data ingestion calculated?
+We define ingested bytes as bytes that we receive, process, and store on your behalf. This includes trace metadata, LLM inputs/outputs, and any other information you explicitly log to Weave, but does not include communication overhead (e.g., HTTP headers) or any other data that is not placed in long term storage. We count bytes as "ingested" only once at the time they are received and stored.
```

Comment:
Some customers think that deleting data will reduce their bill. It may be good to clarify that we bill on "ingested" and not "stored". So deletion does not change change their bill.